resources/views/home.blade.php

```diff
@@ -0,0 +1,15 @@
+@extends('layouts/app')
+@section('content')
+
+<main>
+    <div class="container text-center">
+        <h2 class="m-3 text-light">Remember?</h2>
+        <div>
+            <a href="" class="d-block rounded-lg mb-4 pb-1 bg-light text-decoration-none">
+                <img class="img-fluid rounded-top mb-3" src="{{ asset('images/portrait.jpg') }}" />
+                <h4 class="text-dark  no-gutters ">some song</h4>
+            </a>
+        </div>
```

Comment:
If you're using bootstrap, the [card elements](https://getbootstrap.com/docs/4.4/components/card/) might be something for this case